generation/localizedstringkit/__init__.py

```diff
@@ -99,16 +111,26 @@ def has_changes(*, localized_string_kit_path: str, code_files: List[str]) -> boo
 
     log.info("Determining if localization needs run")
 
-    existing_strings_path = os.path.join(localized_string_kit_path, SOURCE_CODE_FILE_NAME)
+    # Generate current code file paths
+    current_strings_paths = generate_code_strings_file(code_files)
+
+    # Iterate through code files (1 per bundle)
+    for code_file in code_files:
+        existing_strings_path = os.path.join(localized_string_kit_path, "source_strings.m")
```

Comment:
It's probably not worth handling this explicitly for 3 reasons:

1) It's extra effort to maintain.
2) Whoever merges these changes into their repo will be forced to do this anyway
3) If we just check for existing bundles, it will find the difference anyway

Reply:
Updated to drop `source_strings.m` and use a code file name that matches the `.bundle` name